batavia/core/types/PYCFile.js

```diff
@@ -5,6 +5,12 @@ var constants = require('../constants')
  *************************************************************************/
 
 var PYCFile = function(data) {
+
+    if (!Uint8Array.prototype.slice) {
+        Object.defineProperty(Uint8Array.prototype, 'slice', {
+            value: Array.prototype.slice
+        });
```

Comment:
The linter might complain about this semicolon (I don't think we have them elsewhere.